packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -383,6 +550,53 @@ class SKProductWrapper {
       subscriptionPeriod,
       introductoryPrice,
       discounts);
+
+  /// Convert from [SKProductMessage] to [SKProductWrapper]
+  static SKProductWrapper convertFromPigeon(SKProductMessage msg) {
```

Comment:
No action required: These are all a pain.  I guess if pigeon had been used from day one, the generated classes could have been been used.  We also talked about letting pigeon use existing classes too.  That would have really helped here.